src/gui/quotainfo.cpp

```diff
@@ -99,14 +99,14 @@ void QuotaInfo::slotCheckQuota()
     }
 
     const AccountPtr &account = _accountState->account();
-    _job = new PropfindJob(account, account->davUrl(), quotaBaseFolder(), this);
+    _job = new PropfindJob(account, account->davUrl(), quotaBaseFolder(), 0, this);
     _job->setProperties({ QByteArrayLiteral("quota-available-bytes"), QByteArrayLiteral("quota-used-bytes") });
-    connect(_job.data(), &PropfindJob::result, this, &QuotaInfo::slotUpdateLastQuota);
+    connect(_job.data(), &PropfindJob::directoryListingIterated, this, &QuotaInfo::slotUpdateLastQuota);
     connect(_job.data(), &AbstractNetworkJob::networkError, this, &QuotaInfo::slotRequestFailed);
     _job->start();
 }
 
-void QuotaInfo::slotUpdateLastQuota(const QMap<QString, QString> &result)
+void QuotaInfo::slotUpdateLastQuota(const QString &, const QMap<QString, QString> &result)
```

Comment:
If the string isn't used, why add it to the signature?

Reply:
Because thats how connects work

Reply:
https://github.com/owncloud/client/pull/10181#discussion_r993405244